Doc/using/cmdline.rst

```diff
@@ -994,3 +1000,110 @@ Debug-mode variables
    Need Python configured with the :option:`--with-trace-refs` build option.
 
    .. versionadded:: 3.11
+
+.. _using-on-finding-modules:
```

Comment:
importlib finds module. This section is only about sys.path **initialization**: it can be modified afterwards by modifying directly sys.path list. I suggest to rename the link to "sys-path-init".

Reply:
Fair comment, I have amended.